### PR TITLE
Pass print stream to binding providers

### PR DESCRIPTION
**Pull request by gruberrolandvaltech**
This allows binding providers to access the print stream. This is required to output messages to output text.
This is an incompatible API change which should be fine. The latest changes to binding interface also included an incompatible change.

### CODE REVIEW

**Comment by gruberrolandvaltech** [comment]
@lykorian Happy new year! :) Is the code fine for you or do you see room for improvement?

File: src/main/groovy/com/icfolson/aem/groovy/console/components/BindingsPanel.groovy [hunk 1]
```diff
@@ -20,6 +22,7 @@ class BindingsPanel {
 
     @Memoized
     Map<String, BindingVariable> getBindingVariables() {
-        extensionService.getBindingVariables(request)
+        def printStream = new PrintStream(stream, true, CharEncoding.UTF_8)
```
Comment (lykorian) [inline src/main/groovy/com/icfolson/aem/groovy/console/components/BindingsPanel.groovy]: The `stream` variable does not exist

**Comment by lykorian** [comment]
Can you give an example of how this change would be used?

File: src/main/groovy/com/icfolson/aem/groovy/console/components/BindingsPanel.groovy (hunk 1 above)
Comment (gruberrolandvaltech) [reply src/main/groovy/com/icfolson/aem/groovy/console/components/BindingsPanel.groovy]: Thanks, fixed

**Comment by gruberrolandvaltech** [comment]
In our binding we need to print to the output stream so data gets into the response of the script. Currently, we do this by returning a string and using "println" inside script.

Simple example:

println aecu.contentUpgradeBuilder()
     .forResources((String[]) ["/content/we-retail/ca/en"])
     .printPath()
     .run()

But this is bad and hard to understand for users. So we would like to get rid of the "println" at the beginning. For this our binding needs access to the print stream.
I think there is no other possibility right now to write to the run's output area.

**Comment by gruberrolandvaltech** [comment]
Thanks! :)

**Pull request closed without merging**